File: seoyeon/0128/외톨이 알파벳.java
import java.util.*;

class Solution {
    public String solution(String input_string) {
        List<String> answer = new ArrayList<>();
        String temp = "";
        String check = "";

        for (String s : input_string.split("")) {
            if (temp.equals(s)) continue;
            temp = s;

            if (!check.contains(s))
                check += s;
            else if (!answer.contains(s))
                answer.add(s);
        }

        Collections.sort(answer);
        return answer.size() == 0 ? "N" : String.join("", answer);
    }
}

File: chowon/0128/외톨이 알파벳.py
def solution(input_string):
    answer = ''

    for p in range(0,len(input_string)):
        if input_string[p] in answer: continue
        for q in range(p+1,len(input_string)):
            if input_string[p] == input_string[q]:
                if ((q-p) == 1): break
                elif ((q-p) > 1):
                    answer += input_string[p]
                    break

    if len(answer) == 0 : return 'N'
    else: return ''.join(sorted(answer))
